dynamo-zip-store/src/main/scala/dynamo_zip_store/DynamoZipStore.scala

```diff
@@ -0,0 +1,52 @@
+package dynamo_zip_store
+
+
+import cats.effect.IO
+import cats.effect.unsafe.implicits.global
+import fs2.Stream
+import software.amazon.awssdk.services.dynamodb.{DynamoDbAsyncClient, DynamoDbClient}
+import software.amazon.awssdk.services.dynamodb.model.{AttributeValue, GetItemRequest, PutItemRequest}
+import zip_partitioner.FileArchive
+
+import scala.jdk.CollectionConverters.MapHasAsJava
+
+object DynamoZipStore {
+
+  def uploadFilesToDynamoDB(client: DynamoDbAsyncClient, fileArchives: Stream[IO, FileArchive], tableName: String, keyColumnName: String, valueColumnName: String): Stream[IO, PutItemRequest] = {
```

Comment:
maybe it will be better to do a DynamoDestinationConfig case class and have tableName, keyColumnName and valueColumnName (rename to dataColumnName)